examples/resources/p0_okta_group_assignment/resource.tf

```
@@ -1,15 +1,11 @@
-resource "p0_okta_group_assignment_staged" "example" {
-  domain = "example.okta.com"
-}
-
 resource "okta_app_oauth_role_assignment" "p0_group_membership_admin_role_assignment" {
   type      = "GROUP_MEMBERSHIP_ADMIN"
   client_id = okta_app_oauth.p0_api_integration.client_id
 }
 
 resource "p0_okta_group_assignment" "example" {
   depends_on = [
-    p0_okta_group_assignment_staged.example
+    p0_okta_directory_listing.example
   ]
-  domain = p0_okta_group_assignment_staged.example.domain
+  domain = "example.okta.com"
```

this would generally be
```suggestion
  domain = p0_okta_directory_listing.example.domain
```
right?